### PR TITLE
moyo header

### PR DESCRIPTION
**Pull request by andrii-petruk**
- [DEMO LINK] (https://andrii-petruk.github.io/layout_moyo-header/)
- [TEST REPORT LINK] (https://andrii-petruk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,64 @@
...
+  margin: auto 0;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:48]: move repeated colors to variables

File: src/style.css [hunk 2]
```diff
@@ -1 +1,64 @@
...
+.nav__link {
+  padding: 23px 0;
+  display: flex;
+  text-transform: uppercase;
+  line-height: 14px;
+  color: #000;
+  text-decoration: none;
+}
+
+.logo {
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:37]:
```suggestion
.header__logo {
```
isn't this logo an element of header?

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Good job 👍 
Pls, check my comments
Also check design - there is box-shadow for header

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,69 @@
...
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#apple" 
+              class="nav__link is-active">Apple
+            </a>
```
Comment (DarkMistyRoom) [inline src/index.html:29]:
```suggestion
            <a 
              href="#apple" 
              class="nav__link is-active"
            >
              Apple
            </a>
```

Comment (DarkMistyRoom) [reply src/index.html:29]: Please fix it everywhere

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,69 @@
...
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#samsung" class="nav__link">Samsung
+            </a>
```
Comment (DarkMistyRoom) [inline src/index.html:34]:
```suggestion
            <a href="#samsung" class="nav__link">Samsung</a>
```

Comment (DarkMistyRoom) [reply src/index.html:34]: You don't have to place less then 3 attributes on a new line

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,69 @@
...
+          <li class="nav__item">
+            <a 
+              href="#smartphones" class="nav__link">Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#laptops&computers" class="nav__link" 
+              data-qa="hover">Laptops & Computers
+            </a>
```
Comment (DarkMistyRoom) [inline src/index.html:45]:
```suggestion
            <a 
              href="#laptops&computers" 
              class="nav__link" 
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Comment (DarkMistyRoom) [reply src/index.html:45]: but if you do it, place each attribute on a separate line

File: src/index.html [hunk 6]
```diff
@@ -7,8 +7,69 @@
...
+            <a 
+              href="#samsung" class="nav__link">Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#smartphones" class="nav__link">Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (DarkMistyRoom) [inline src/index.html:41]: Add empty lines between multiline sibling blocks of HTML

File: src/style.css [hunk 7]
```diff
@@ -1 +1,66 @@
...
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body,
+ul {
+  margin: 0;
+  --color: #00acdc;
```
Comment (DarkMistyRoom) [inline src/style.css:11]: move `--color` to the `:root`

File: src/style.css [hunk 8]
```diff
@@ -1 +1,66 @@
...
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
```
Comment (DarkMistyRoom) [inline src/style.css:26]: why do you need to do it twice? `margin: 0;`

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**